x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/details_page/components/package_policies/package_policies_table.tsx

```diff
@@ -109,29 +116,62 @@ export const PackagePoliciesTable: React.FunctionComponent<Props> = ({
           defaultMessage: 'Name',
         }),
         render: (value: string, packagePolicy: InMemoryPackagePolicy) => (
-          <EuiLink
-            title={value}
-            {...(canReadIntegrationPolicies
-              ? {
-                  href: getHref('edit_integration', {
-                    policyId: agentPolicy.id,
-                    packagePolicyId: packagePolicy.id,
-                  }),
-                }
-              : { disabled: true })}
-          >
-            <span className="eui-textTruncate" title={value}>
-              {value}
-            </span>
-            {packagePolicy.description ? (
-              <span>
-                &nbsp;
-                <EuiToolTip content={packagePolicy.description}>
-                  <EuiIcon type="help" />
-                </EuiToolTip>
-              </span>
-            ) : null}
-          </EuiLink>
+          <EuiFlexGroup gutterSize="s" alignItems="center">
+            <EuiFlexItem data-test-subj="PackagePoliciesTableName" grow={false}>
+              <EuiLink
+                title={value}
+                {...(canReadIntegrationPolicies
+                  ? {
+                      href: getHref('edit_integration', {
+                        policyId: agentPolicy.id,
+                        packagePolicyId: packagePolicy.id,
+                      }),
+                    }
+                  : { disabled: true })}
+              >
+                <span className="eui-textTruncate" title={value}>
+                  {value}
+                </span>
+                {packagePolicy.description ? (
+                  <span>
+                    &nbsp;
+                    <EuiToolTip content={packagePolicy.description}>
+                      <EuiIcon type="help" />
+                    </EuiToolTip>
+                  </span>
+                ) : null}
+              </EuiLink>
+            </EuiFlexItem>
+            {canUseMultipleAgentPolicies &&
+              canReadAgentPolicies &&
+              canWriteIntegrationPolicies &&
```

Comment:
```suggestion
              canReadIntegrationPolicies &&
```

I think it should be read integration policies according to the issue: https://github.com/elastic/kibana/issues/182219